cardano-crypto-class/src/Cardano/Crypto/VRF/Class.hs

```diff
@@ -105,31 +103,39 @@ class ( Typeable v
   type Signable v :: Type -> Constraint
   type Signable c = Empty
 
+  type OutputVRF v :: Type
+  type OutputVRF v = ByteString
```

Comment:
We never use the flexibility offered here, and indeed for things like `CertifiedVRF` in this module we have to fix it. So I would say let's just remove it; all VRF classes must output a ByteString.

Also `sizeOutputVRF` isn't well-defined if `OutputVRF` can be different types.

Reply:
agreed.